routing_common/vehicle_model.hpp

```diff
@@ -37,8 +37,11 @@ class IVehicleModel
 
   virtual bool IsOneWay(FeatureType const & f) const = 0;
 
-  /// @returns true iff feature |f| can be used for routing with corresponding vehicle model.
+  /// @returns true if feature |f| can be used for routing with corresponding vehicle model.
```

Comment:
should I change back if to iff and then do some separated pull request to fix spelling in comments?